messaging/firebase-messaging-sw.js

```diff
@@ -42,8 +42,8 @@ const messaging = firebase.messaging();
 // If you would like to customize notifications that are received in the
 // background (Web app is closed or not in browser focus) then you should
 // implement this optional method.
-// [START background_handler]
-messaging.setBackgroundMessageHandler(function(payload) {
+// [START onBackgroundMessage]
```

Comment:
Nit: Could you use `on_background_message` for this include tag just for consistency?